globals.gd

```diff
@@ -1300,6 +1346,19 @@ var sexuality_images = {
 	futa_3 = load("res://files/aric_expansion_images/sexuality_icons/futa_3.png"),
 }
 
+var playerspecs = {
```

Comment:
Adding this variable to the mod replaces the vanilla version, but is never used. It should be removed unless there is a reason to keep it.

Reply:
Fixed